examples/Common patterns for connectors/pagination/page_number/connector.py

```diff
@@ -1,6 +1,7 @@
 # This is a simple example for how to work with page-number-based pagination for a REST API.
 # It defines a simple `update` method, which upserts retrieved data to a table named "item".
-# THIS EXAMPLE IS THE JUST FOR UNDERSTANDING AND NEEDS MODIFICATION OF A REAL API URL TO MAKE IT WORK.
+# THIS EXAMPLE IS THE JUST FOR UNDERSTANDING AND NEEDS FIVETRAN-API-PLAYGROUND
```

Comment:
Something like:
This example is to help you understand concepts using dummy data. It requires the fivetran-api-playground package to run.